language/use-cases/applying-llms-to-data/bigquery_dataframes_ml_drug_name_generation.ipynb

```diff
@@ -0,0 +1,989 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    CONNECTION = ""  # @param {type:"string"}
</pre><p>Can we put in a default connection name here, like in the other BQ DF notebook? e.g. "bqdf-ml-drugname", "bqdf-llm", etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
put a default `CONNECTION = "bigframes-ml"`